automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -136,66 +136,74 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
           val secondApiCall =
             Leonardo.cluster.getRuntime(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName)
           firstApiCall.auditInfo.dateAccessed should be < secondApiCall.auditInfo.dateAccessed
+          Leonardo.cluster.stopRuntime(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName)
         }
       }
     }
 
-    Seq(Python3).foreach { kernel =>
-      s"should preinstall google cloud subpackages for ${kernel.string}" in { runtimeFixture =>
-        withWebDriver { implicit driver =>
-          withNewNotebook(runtimeFixture.runtime, kernel) { notebookPage =>
-            //all other packages cannot be tested for their versions in this manner
-            //warnings are ignored because they are benign warnings that show up for python2 because of compilation against an older numpy
-            notebookPage.executeCell(
-              "import warnings; warnings.simplefilter('ignore')\nfrom google.cloud import bigquery\nprint(bigquery.__version__)"
-            ) shouldBe Some("1.23.1")
-            notebookPage.executeCell("from google.cloud import datastore\nprint(datastore.__version__)") shouldBe Some(
-              "1.10.0"
-            )
-            notebookPage.executeCell("from google.cloud import storage\nprint(storage.__version__)") shouldBe Some(
-              "1.23.0"
-            )
-          }
-        }
-      }
-    }
-
+    //TODO: uncomment the test when selenium issue is fixed
```

Comment:
Do we know why these tests fail but not others? (Is it an ordering issue?)

Reply:
I don't know what the issue is...the symptom is as if chrome didn't start up properly for these tests. Even for the tests that are already enabled, it seems we do have some flakiness that some specs might have the selenium error still...I'm hoping to looking into this particular issue more (but I'm quite stuck at the moment regarding this) once we have most the non-selenium issue tests enabled.